pkg/api/fileshare.go

```diff
@@ -0,0 +1,314 @@
+package api
+
+import (
+	"encoding/json"
+	"fmt"
+
+	log "github.com/golang/glog"
+	c "github.com/opensds/opensds/pkg/context"
+	"github.com/opensds/opensds/pkg/controller/client"
```

Comment:
has to be changed 
"github.com/opensds/opensds/pkg/filesharecontroller/client"
	pb "github.com/opensds/opensds/pkg/model/fileshareproto"